linux_os/guide/services/ssh/ssh_server/sshd_enable_gssapi_auth/rule.yml

```diff
@@ -0,0 +1,32 @@
+documentation_complete: true
+
+title: 'Enable GSSAPI Authentication'
+
+description: |-
+    Sites setup to use Kerberos or other GSSAPI Authenticaion require setting
+    sshd to accept this authentication.
+    To enable GSSAPI authentication, add or correct the following line in the
+    <tt>/etc/ssh/sshd_config</tt> file:
+    <pre>GSSAPIAuthentication yes</pre>
+
+rationale: |-
+    GSSAPI authentication is used to provide additional authentication mechanisms to
+    applications.
```

Comment:
Please update the rationale to be something similar:

```
    Kerberos authentication for SSH is often implemented using GSSAPI. If Kerberos
    is enabled through SSH, the SSH daemon provides a means of access to the
    system's Kerberos implementation. Vulnerabilities in the system's Kerberos
    implementations may be subject to exploitation.

    For enterprises, Kerberos is often enabled and used with GSSAPI for 
    centralized user account management which may necessitate enabling of
    GSSAPI functionality in SSH. 
```

Reply:
I'll switch this out and force push shortly.